packages/vscode-extension/src/Logger.ts

```diff
@@ -24,7 +24,12 @@ const logger = {
   },
 
   error(message: string, ...args: any[]) {
-    outputChannel.error(message, ...args);
+    if (args.length > 0 && args[args.length - 1] instanceof Error) {
+      const error = args[args.length - 1] as Error;
```

Comment:
is the as Error necessary, I think `&& args[args.length - 1] instanceof Error` should cause type cast by itself

Reply:
no, apparently cannot infer the type this way. If I first stored it in variable and only have `instanceof` there, it'd work. We actually don't need to make it of Error type, as the logger then accepts "any" as input. But this way it is more explicit.